### ISSUE TITLE
1.9 Accessibility information text

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4852]
- #4694 will add a place for accessibility information.
- This will then need to be updated with each release.

So first, the information for 1.9 needs to be drafted!

- [ ] link to A11y project
- [ ] link to A11y roadmap
- [ ] link to contact page
- [ ] prominent link to Sitemap
- [ ] link to Janeway site for more info (? vpat on Janeway site?)

As part of this, may need to reorganise the A11y project board, so that say ''known bugs" are clearly listed in a single view, like how releases are.

Need to consider how this is translated.  If we are making a statement about Janeway accessibility in English, do we need to be clear that a translation by someone else  and give access to the original?

**Comment by StephDriver** [comment]
Note. Consider the inaccessibility of Material navigation and how best to redirect users not navigating by mouse.  This should be stated on the accessibility page for material theme, but perhaps something should also be in the navigation that explains the workaround?

It can be navigated by keyboard but only if one uses different controls.  Level 1 uses tab, but the dropdowns once open, use up/down arrows. This is intrinsic to materialize and so far beyond scope to fix.
Similarly, updates have given better screen reader support, but it is not fully accessible as the behaviour is not as expected.  It can be navigated, but only with the 'right' keys.

**Comment by StephDriver** [comment]
## Page Features from User Stories

I am considered two particular kinds of user for this page.  Primarily there will be site-visitors who themselves have accessibility needs. Secondly there will be accessibility leads who are evaluating whether the site can be used by their clients, staff, students.

From this have found 6 key features for this page. 

| Library A11y Lead                          | Visitor with A11y needs                        | Feature                                            |
| ------------------------------------------ | ---------------------------------------------- | -------------------------------------------------- |
| Is site compliant with X standard?         | Can I use this site?                           | VPAT.                                              |
|                                            | Why does X page go wrong with my system?       | Known bugs                                         |
| Can our students/staff use this site?      | Should I use A11y mode?                        | Recommendations for features improved by A11y mode |
|                                            | What does A11y mode mean? Will I lose content? | A11y Mode explainer                                |
| What is the direction of travel (on A11y)? | Is accessiblity improving?                     | A11y Project explainer and link                    |
| Who do I contact about A11y issues?        | Who do I contact about A11y issues?            | Contact form link                                  |

**Comment by ajrbyers** [comment]
DONE!
